contract.md

```diff
@@ -4,6 +4,10 @@
 
 _At a high level, what do you most want to get out of this project, and why?_
 
+At a high level, I would like to feel that I can apply the skills I have learned from this class to design and implement other DSLs after having finished this project. While this might not be something that I need to do, I would appreciate having a clearer sense of what the process is like for implementing a DSL of my own design than I currently have. This would allow me to come away from this class feeling that I can create things which I appreciate the value of, and thus I will have a new skill that I can later use if needed.
+
+Many classes require doing a predefined final project or exams to test skills and though these can be useful methods for evaluating the students' grasp of key concepts, it does not provide the sense of satisfaction and feeling of truly understanding the value of knowing those concepts. Also, this project allows us to make something that is much more abstract and that is why I appreciate the fact that this project is from scratch, with us designing our language, choosing the concepts we would like to work on, and setting our own learning expectations.
```

Comment:
I feel the same way! I think this project is a great way to show our learning and skills from the class but also gain the satisfaction of building something from scratch.